app/src/main/java/com/a1ishka/bankaccount/data/repository/AccountRepositoryImpl.kt

```diff
@@ -0,0 +1,67 @@
+package com.a1ishka.bankaccount.data.repository
+
+import com.a1ishka.bankaccount.domain.AccountDao
+import com.a1ishka.bankaccount.data.accountData
+import com.a1ishka.bankaccount.data.entity.AccountEntity
+import com.a1ishka.bankaccount.domain.repository.AccountRepository
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flatMapConcat
+import kotlinx.coroutines.flow.flowOf
+
+// for multiple datasources (demo with ready lists)
```

Comment:
in this task you have the only datasource. No need to add extra work and lines of code

Reply:
cleaned